### PR TITLE
Expose target in bldr job status

### PR DESCRIPTION
**Pull request by eeyun**
Signed-off-by: Ian Henry <ihenry@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-worker/src/server.rs [hunk 1]
```diff
@@ -39,6 +39,9 @@ use crate::log_forwarder::LogForwarder;
 use crate::network::NetworkNamespace;
 use crate::runner::{studio, RunnerCli, RunnerMgr};
 
+/// Interval for main thread to check cancel status
+pub const BUILD_CANCEL_WAIT_SECS: u64 = 15;
```
Comment (eeyun) [inline components/builder-worker/src/server.rs]: I'm.. very confused as to what's going on here. It looks like my indexes might be boogered up

Comment (chefsalim) [reply components/builder-worker/src/server.rs]: rustfmt issue?

**Comment by eeyun** [comment]
I rebased this to correct something strange I was seeing. Now I need to retest.

**Comment by eeyun** [comment]
Phew everything still working.

**Pull request merged**